src/workbooks/governance/workbook.json

```
@@ -5333,373 +5672,387 @@
+                                "formatter": 7,
+                                "formatOptions": {
+                                  "linkTarget": "GenericDetails",
+                                  "linkIsContextBlade": true
+                                }
+                              },
+                              {
+                                "columnMatch": "policyAssignmentScope",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "policyAssignmentDefinitionId",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "policyAssignmentNotScopes",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "definitionId",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "numberOfPolicies",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "numberOfGroups",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "isRegulatoryInitiative",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "isRPMode",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "policySetDefinitionId",
+                                "formatter": 5
+                              },
+                              {
+                                "columnMatch": "details",
+                                "formatter": 7,
+                                "formatOptions": {
+                                  "linkTarget": "CellDetails",
+                                  "linkLabel": "🔍 View Details",
+                                  "linkIsContextBlade": true
+                                }
+                              }
+                            ],
+                            "filter": true,
+                            "sortBy": [
+                              {
+                                "itemKey": "category",
+                                "sortOrder": 2
+                              }
+                            ],
+                            "labelSettings": [
+                              {
+                                "columnId": "policyAssignmentId",
+                                "label": "Assignment Id"
```

Microsoft always writes identifier as "ID" in text and uses "Id" in variable names.
```suggestion
                                "label": "Assignment ID"
```